classes.py

```diff
@@ -5,15 +5,59 @@
 
 #define the Dice class
 class Dice:
+    """
+    A class used to represent the five dice
+    
+    ...
+    
+    Attributes
+    ----------
+    current_throw : list
+        List of integers of the five dice
+        
+    Methods
+    ----------
+    roll(throw)
+        Assigns random integers to current_throw 
+    """
+    
     def __init__(self):
-        self.current_throw = [randint(1, 6) for i in range(0, 5)]       
+        self.current_throw = [randint(1, 6) for i in range(0, 5)]   
```

Comment:
hi! just out of interest, can I ask why you have the exact same for loops two times? :)

Reply:
I don't know why Github sees it as a difference. It is actually just one line (the green line)